source/index.rst

```diff
@@ -8,6 +8,8 @@
 FIRST Robotics Competition Control System
 ===========================================
 
+.. warning:: Installers before ``2021.2.2`` **and** ``3/24/2021`` are unable to install VS Code! Please `redownload <https://github.com/wpilibsuite/allwpilib/releases>`__ the ``2021.2.2`` (or greater) release. We are greatly sorry for this inconvenience. Please see :doc:`Known Issues </docs/yearly-overview/known-issues>` for more information.
```

Comment:
As discussed we do not need to add a warning to the homepage other communication avenues will do.